RecoHGCal/TICL/python/HADStep_cff.py

```diff
@@ -38,32 +38,37 @@
     ,filteredLayerClustersHAD
     ,ticlTrackstersHAD)
 
-filteredLayerClustersHFNoseHAD = _filteredLayerClustersProducer.clone(
+# HFNOSE CLUSTER FILTERING/MASKING
+
+filteredLayerClustersHFNoseHAD = filteredLayerClustersHAD.clone(
     min_cluster_size = 2, # inclusive
     algo_number = 9,
     iteration_label = "HADn",
     LayerClusters = 'hgcalLayerClustersHFNose',
-    LayerClustersInputMask = "hgcalLayerClustersHFNose:InitialLayerClustersMask"
+    LayerClustersInputMask = "ticlTrackstersHFNoseTrk"
 )
 
-ticlTrackstersHFNoseHAD = _trackstersProducer.clone(
+# HFNOSE CA - PATTERN RECOGNITION
+
+ticlTrackstersHFNoseHAD = ticlTrackstersHAD.clone(
     detector = "HFNose",
     layer_clusters = "hgcalLayerClustersHFNose",
     layer_clusters_hfnose_tiles = "ticlLayerTileHFNose",
-    original_mask = "hgcalLayerClustersHFNose:InitialLayerClustersMask",
+    original_mask = "ticlTrackstersHFNoseTrk", #"ticlTrackstersHFNoseTrk",
     filtered_mask = "filteredLayerClustersHFNoseHAD:HADn",
     seeding_regions = "ticlSeedingGlobalHFNose",
     time_layerclusters = "hgcalLayerClustersHFNose:timeLayerCluster",
-    # For the moment we mask everything w/o requirements since we are last
+    # eid_graph_path: might as well have a separately trained model for nose
     pluginPatternRecognitionByCA = dict(
        pid_threshold = 0.,
        skip_layers = 1,
        min_layers_per_trackster = 5,
        min_cos_theta = 0.866,    # ~30 degrees
-       min_cos_pointing = 0.819, # ~35 degrees
-       max_delta_time = -1,
+       min_cos_pointing = 0.866, # ~30 degrees
+       max_delta_time = -1
+       # eid_n_layers = 8
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
It might be uncommented in the future, as the hfnose only has 8 layers. But I can remove it for now.